spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -428,6 +577,45 @@ function refundTokens(packet: Packet) {
 }
 ```
 
+```typescript
+// revertInFlightChanges reverts the receive packet and send packet
+// that occurs in the middle chains during a packet forwarding
+// If an error occurs further down the line, the state changes
+// on this chain must be reverted before sending back the error acknowledgement
+// to ensure atomic packet forwarding
+function revertInFlightChanges(sentPacket: Packet, receivedPacket: Packet) {
+  forwardEscrow = channelEscrowAddresses[sentPacket.sourceChannel]
```

Comment:
Shouldn't this be `forwardEscrow = channelForwardingAddresses[receivedPacket.destChannel]` instead?